src/autocomplete/dataset.js

```diff
@@ -93,7 +93,7 @@ _.mixin(Dataset.prototype, EventEmitter, {
     var renderArgs = [].slice.call(arguments, 2);
     this.$el.empty();
 
-    hasSuggestions = suggestions && suggestions.length;
+    hasSuggestions = suggestions && suggestions.length > 0;
```

Comment:
Best way is to provide a test for this, because:
```js
var suggestions = [];
var hasSuggestions = suggestions && suggestions.length;
var isEmpty = !hasSuggestions;
console.log(isEmpty);
> true
```

So it seems the current behavior is correct, but maybe in some situations suggestions is not an array (and that's what I need to understand)